From 53f4c2e82375eb8244963e7aff61542869cc493e Mon Sep 17 00:00:00 2001
From: Przemyslaw Czarnota <przemyslawx.czarnota@intel.com>
Date: Mon, 11 Feb 2019 09:53:50 +0100
Subject: [PATCH] uapi: linux: add tc_colmark.h to exported headers

When UAPI headers were installed tc_colmark.h was missing.
This fixes this issue.

Signed-off-by: Przemyslaw Czarnota <przemyslawx.czarnota@intel.com>
---
 include/uapi/linux/tc_act/Kbuild | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/tc_act/Kbuild b/include/uapi/linux/tc_act/Kbuild
index e3db7403296f..935518f8887f 100644
--- a/include/uapi/linux/tc_act/Kbuild
+++ b/include/uapi/linux/tc_act/Kbuild
@@ -1,5 +1,6 @@
 # UAPI Header export list
 header-y += tc_csum.h
+header-y += tc_colmark.h
 header-y += tc_defact.h
 header-y += tc_gact.h
 header-y += tc_ipt.h
